src/NuGet.Clients/NuGet.PackageManagement.UI/Options/AddMappingDialog.xaml.cs

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Windows.Automation.Peers;
+using System.Windows.Automation;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using Microsoft.ServiceHub.Framework;
+using Microsoft.VisualStudio.PlatformUI;
+using NuGet.PackageManagement.UI;
+using NuGet.VisualStudio;
+using NuGet.VisualStudio.Common;
+using NuGet.VisualStudio.Internal.Contracts;
+using Task = System.Threading.Tasks.Task;
+
+namespace NuGet.Options
+{
+    public partial class AddMappingDialog : DialogWindow
+    {
+        public ICommand HideDialogCommand { get; set; }
+
+        public ICommand AddMappingCommand { get; set; }
+
+        public ItemsChangeObservableCollection<PackageSourceViewModel> SourcesCollection { get; private set; }
+
+        private IReadOnlyList<PackageSourceContextInfo> _originalPackageSources;
+
+        private PackageSourceMappingOptionsControl _parent;
+
+        public AddMappingDialog(PackageSourceMappingOptionsControl parent)
+        {
+            _parent = parent;
+
+            // The use of a `JoinableTaskFactory` is not relevant as `CanExecuteChanged` will never be raised on this command.
+#pragma warning disable VSTHRD012 // Provide JoinableTaskFactory where allowed
+            HideDialogCommand = new DelegateCommand(ExecuteHideDialog);
```

Comment:
No need for JTF here since the command is always visible, but the analyzer isn't aware of that.

Reply:
You left a comment on the PR, resolving the thread.